README.md

```diff
@@ -6,73 +6,72 @@ https://firecloud.dsde-dev.broadinstitute.org/
 
 https://firecloud.dsde-staging.broadinstitute.org/
 
-## Getting Started
+## Technologies
 
-This project runs in a Docker container. Docker is available here:
+[ClojureScript](https://github.com/clojure/clojurescript) is used for the UI.
 
-https://www.docker.com/
+We use the [Leiningen](http://leiningen.org/) build tool to manage ClojureScript dependencies.
 
-[ClojureScript](https://github.com/clojure/clojurescript) is used for the UI.
-We use the [Leiningen](http://leiningen.org/) build tool.
 The code incorporates usage of [react-cljs](https://github.com/dmohs/react-cljs) which is 
 a ClojureScript wrapper for [React](https://facebook.github.io/react/).
 
-For authentication, you will need to create a Google web application client ID from here:
+Figwheel replaces the running JavaScript within the page so changes are visible without a browser reload. More information [here](https://github.com/bhauman/lein-figwheel). [This video](https://www.youtube.com/watch?v=j-kj2qwJa_E) gives some insight into the productivity gains available when using this technology (up to about 15:00 is all that is necessary).
+
+## Getting Started
+
+Start with a [docker](https://www.docker.com/) environment.
+
+You will need to create a Google web application client ID from here:
```

Comment:
I would suggest some more documentation around this - there are a lot of gotchas.

* Create a new OAuth client ID under APIs & Auth: Credentials from the Google Developer's Console
* Name it such that it is clear this is a developer id and not an application id
* Add authorized javascript origins
  * http://dhost
  * http://192.168.99.100 (or your docker machine's IP(s)
  * http://localhost:8000 (or any other versions of that depending on your app environment)
* Hit "Save" for each origin addition, and hit it again to save the final configuration. You'll know you've finished when you're redirected back to the Credentials list page.
* Make sure your /etc/hosts file has appropriate mappings for your docker machine(s):
  * 192.168.99.100 points to dhost